common/src/types.rs

```diff
@@ -4,153 +4,3 @@
 pub type Address = String;
 pub type InboxId = String;
 pub type WalletAddress = String;
-
```

Comment:
moved to `xmtp_proto/`, since protocol-specific types seem to fit in better there, then a general 'common' crate aimed more at rusty-things